pkg/dsl/process/handle.go

```diff
@@ -110,11 +133,20 @@ func (ctx *templateContext) BaseContextFile() string {
 		buff += ConfigFieldName + ": " + string(bt)
 	}
 
+	if len(ctx.requiredSecrets) > 0 {
+		for _, s := range ctx.requiredSecrets {
+			data, _ := json.Marshal(s.Data)
+			buff += s.ContextName + ":" + string(data) + "\n"
+		}
+	}
+
+	if ctx.outputSecretName != "" {
+		buff += fmt.Sprintf("%s:\"%s\"", OutputSecretName, ctx.outputSecretName)
```

Comment:
This is not included in design, when will it be used?

And what about there's more than one secret?

Reply:
As we talked just now. `outputSecretName` will be kept to indicate the secret name and its namespace will be stored globally later. Then consumer will know what secret could be consumed and  we can check whether there are conflicts.

Reply:
There will be only one output secret name for one cloud resource component definition.

Reply:
remove outputSecretName related code. And will extend it for cross-namespace secret in the future.